src/applications/edu-benefits/1990e/containers/IntroductionPage.jsx

```diff
@@ -3,114 +3,153 @@ import { focusElement } from 'platform/utilities/ui';
 import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+import WizardContainer from '../../wizard/containers/WizardContainer';
+import {
+  WIZARD_STATUS_NOT_STARTED,
+  WIZARD_STATUS_COMPLETE,
+} from 'applications/static-pages/wizard';
+import environment from 'platform/utilities/environment';
 
 export class IntroductionPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      wizardStatus:
+        sessionStorage.getItem('wizardStatus') || WIZARD_STATUS_NOT_STARTED,
+      isProd: environment.isProduction(),
+    };
+    this.setWizardStatus = this.setWizardStatus.bind(this);
+  }
   componentDidMount() {
     focusElement('.va-nav-breadcrumbs-list');
   }
+
+  setWizardStatus(value) {
+    sessionStorage.setItem('wizardStatus', value);
+    this.setState({ wizardStatus: value });
+  }
+
   render() {
+    const { wizardStatus, isProd } = this.state;
+    const shouldSubwayMapShow =
+      isProd || wizardStatus === WIZARD_STATUS_COMPLETE;
+    const shouldWizardShow = !isProd && wizardStatus !== WIZARD_STATUS_COMPLETE;
```

Comment:
Okay so the reason those tests are failing is because e2e tests are run with a BUILDTYPE of VAGOVPROD - so according to these conditions which we render the components based on the environment, these will never show up in e2e.  The proper way to test this and set this up would have been to utilize a feature flag.

Reply:
Got a link to how to set those up or  is this it? https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-toggles/

Reply:
Nope thats the link :)

Reply:
Okay feature flag is set. Now where would I set whether this should show in prod?

Reply:
you should be able to mock a redux state/ prop - mock the piece for this flag to true

Reply:
I've got something like this in play:

```
export const showEduBenefits1990EWizard = state =>
  toggleValues(state)[FEATURE_FLAG_NAMES.showEduBenefits1990EWizard];
```

```
  const state = {
    featureToggles: {
      showEduBenefits1990EWizard: true,
    },
  };
  const shouldWizardShow = showEduBenefits1990EWizard(state);

```

But the test still can't find that element:

```
Timed out while waiting for element <.wizard-container> to be present for 2000 milliseconds.
```